docs/running.md

```diff
@@ -1,6 +1,10 @@
 # Running services
 
 Grid consists of many micro-services. There are a few different ways to start them.
+Note that some services don't compile until you hit them for the first time, once it's running you could try:
+
+for x in 900{1..9}; do curl http://localhost:$x; done
+for x in 901{0..11}; do curl http://localhost:$x; done
```

Comment:
I'd remove these two lines here, just because you don't need all the services running most of the time. I've had loads of problems with sbt when running grid, and I think the more services get complied the harder it is. Maybe it might be useful to add a separate note for thrall because that's what people are using mostly?